examples/widgets/rectangle_selector.py

```diff
@@ -7,7 +7,7 @@
 the button.  This class gives click- and release-events and also draws
 a line or a box from the click-point to the actual mouseposition
 (within the same axes) until the button is released.  Within the
-method 'self.ignore()' it is checked whether the button from eventpress
+method ``self.ignore()`` it is checked whether the button from eventpress
```

Comment:
This should really be `within :meth:\`.ignore\`, no?  (not that it really matters here as this is not an API doc, but still...).

Reply:
Well, possibly. But there are multiple `.ignore` methods and I was too lazy to find out which one to link. IMHO it's good enough for now.